src/spikeinterface/preprocessing/tests/test_decimate.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from pathlib import Path
+
+import itertools
+from spikeinterface import NumpyRecording
+from spikeinterface.core import generate_recording
+from spikeinterface.preprocessing.decimate import DecimateRecording
+import numpy as np
+
+
+@pytest.mark.parametrize("decimation_frame_start", [0, 1, 9, 10, 11, 200])
+@pytest.mark.parametrize("decimation_frame_step", [1, 9, 10, 11, 200])
```

Comment:
A couple of comments:

- shouldn't `decimation_frame_start` < `decimation_frame_step` be enforced?
- shouldn't we assert that `decimation_frame_start`  < `num_samples`?